tests/_files/source_with_multiline_constant_return.php

```diff
@@ -485,10 +485,30 @@ public function unaryMinusNowdoc(): bool
     {
         return
             -
+            (int)
             <<<'EOF'
                 foo
                 foo
                 EOF
         ;
     }
+
+    public function concatWithVar(): string
+    {
+        $var1 = 'start';
+
+        $var1 =
+            'right'
+            .
+            $var1
+        ;
+
+        $var1 =
+            $var1
+            .
+            'left'
```

Comment:
instead of string use nowdoc here, I expect a wrong line then

Reply:
We should use both then

Reply:
I would say nowdoc is enough as we test "AST processing" is used instead of "node stard/end line"

Reply:
:+1: